Source/WebCore/svg/SVGStyleElement.idl

```diff
@@ -31,4 +31,6 @@
      attribute [AtomString] DOMString type;
      attribute [AtomString] DOMString media;
      [Reflect] attribute DOMString title;
+
+     readonly attribute StyleSheet sheet;
```

Comment:
`StyleSheet?` per the spec:
https://www.w3.org/TR/cssom-1/#the-linkstyle-interface

Reply:
oh, good point.

Reply:
Also fixing `HTMLStyleElement` for completeness as we discussed on Slack.